core/src/com/unciv/ui/tilegroups/CityButton.kt

```diff
@@ -155,6 +159,52 @@ class CityButton(val city: CityInfo, internal val tileGroup: WorldTileGroup, ski
         parent.addAction(floatAction)
     }
 
+    private fun getPopulationGroup(showGrowth: Boolean): Group {
+        val group = Group()
+
+        val populationLabel = city.population.population.toString().toLabel()
+
+        group.addActor(populationLabel)
+
+        val groupHeight = 25f
+        var groupWidth = populationLabel.width
+        if (showGrowth) groupWidth += 10f
+        group.setSize(groupWidth, groupHeight)
+
+        if (showGrowth) {
+            var growthPercentage = city.population.foodStored / city.population.getFoodToNextPopulation().toFloat()
+            if (growthPercentage < 0) growthPercentage = 0.0f
+
+            val growthBar = ImageGetter.getProgressBarVertical(2f, groupHeight, growthPercentage,
```

Comment:
In case of starvation how about showing a full RED bar? This way it could be easier to spot which cities have negative growth.

Reply:
As for the growth bar, maybe use the secondary color of the civ (or a less _flashy_ green?) when the city is growing?
I think that with some civs colors the light green could look weird:
![off3](https://user-images.githubusercontent.com/17114100/71821452-7dabf000-3092-11ea-8c12-85f6b47c5304.png)